extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufConfig.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.data.input.opencensus.protobuf;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.annotations.VisibleForTesting;
+
+public class OpenCensusProtobufConfig
+{
+
+  @JsonProperty
+  private String resourceLabelPrefix = "resource.";
```

Comment:
while we're at it, we might as well add a prefix config for metric labels as well.
It would also be useful to configure the "metric name" dimensions as well. Our current choice of `name` is probably not the greatest and it could potentially clash with metric labels.

Reply:
@xvrl Do you mean similar prefix behaviour for `metric.getMetricDescriptor().getLabelKeys`?

Reply:
yes, two things:
- we'd like to have a prefix for metric.getMetricDescriptor().getLabelKeys(), which can be empty by default.
- we'd like to allow the user to override the `NAME` constant for the metric name

Reply:
Done.